index.html

```diff
@@ -150,5 +150,36 @@ <h3 class="framework-text">Skills</h3>
                 <li>Terminal</li>
             </ul>
         </section>
+        <section>
+            <p class="text-form">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form>
```

Comment:
- [OPTIONAL] I will highly recommend you to add `action` and `method` attribute in form tag to  integrate with `formspree`. It is the major requirement of this milestone and the previous reviewer has missed checking for it.
so consider applying this change. your form data should get submitted to formspree. kindly click on the following link to get the unique form id.
  -  Collect data by using the [Formspree service](https://formspree.io/html).
  - `action` value should be similar to `https://formspree.io/f/{form_id}`
  - `method` value equals `POST`Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).


action value should be similar to https://formspree.io/f/{form_id}


method value equals POST